lib/TokenManager.ts

```diff
@@ -316,59 +320,36 @@ function renew(sdk, tokenMgmtRef, storage, key) {
     return Promise.reject(e);
   }
 
-  // Remove existing autoRenew timeouts
-  clearExpireEventTimeoutAll(tokenMgmtRef);
+  // Remove existing autoRenew timeout
+  clearExpireEventTimeout(tokenMgmtRef, key);
 
   // A refresh token means a replace instead of renewal
-
   // Store the renew promise state, to avoid renewing again
-  // Renew/refresh all tokens in one process
-  tokenMgmtRef.renewPromise[key] = sdk.token.renewTokens({
-    scopes: token.scopes,
-  })
-    .then(function(freshTokens) {
-      // store and emit events for freshTokens
+  tokenMgmtRef.renewPromise[key] = sdk.token.renew(token)
```

Comment:
sdk.token.renew ([oidc/renewToken](https://github.com/okta/okta-auth-js/blob/master/lib/oidc/renewToken.ts#L18)) only handles renew in the hidden iframe way. This change should still keep the renew with refresh approach.